opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -114,9 +113,9 @@ impl LoggerProvider {
             let mut errs = vec![];
             for processor in &self.inner.processors {
                 if let Err(err) = processor.shutdown() {
-                    otel_warn!(
-                        name: "logger_provider_shutdown_error",
-                        error = format!("{:?}", err)
+                    otel_error!(
```

Comment:
Same as https://github.com/open-telemetry/opentelemetry-rust/pull/2184/files#r1793832380, this feels redundant, given Result will carry this info already.

Reply:
https://github.com/open-telemetry/opentelemetry-rust/pull/2184#discussion_r1793852851

Reply:
Changed this to debug now.